server/migrations/20230304023314-unique-discorduserid.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up (queryInterface, Sequelize) {
+    queryInterface.changeColumn('users', 'discordUserId', {
+      type: Sequelize.STRING,
+      allowNull: false,
+      unique: true,
+    });
+  },
+
+  async down (queryInterface, Sequelize) {
+    queryInterface.changeColumn('users', 'discordUserId', {
+      type: Sequelize.STRING,
+      allowNull: false,
+      unique: false,
+    });
+  }
+};
```

Comment:
Changes to the schema require multiple migrations (you cannot re-run a previous migration without potentially destroying data).  This change ensures that the `users.discord_user_id` is unique.